.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [12.14.1]
```

Comment:
Is there any reason to not use 12.x (it makes sure we always build against latest patch of v12)?

Reply:
Because of latest release of vscode.

Electron 9.0 Update#
In this milestone, we finished the exploration to bundle Electron 9 into VS Code. This is a major Electron release and comes with Chromium 83.0.4103.122 and Node.js 12.14.1.

I thought that we stick to this version. If it si not a problem we could bump to use alway the latest 12.

Reply:
I think we are fine always depending on latest 12.x, the minor updates are backwards compatible.